README.md

```diff
@@ -1,5 +1,7 @@
 # Charmed Kafka ROCK
 
+[![Release](https://github.com/canonical/charmed-kafka-rock/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-rock/actions/workflows/publish.yaml)
+
```

Comment:
In this case the `build ROCK` CI only builds, it doesn't test. Since testing will need the zookeeper rock, we can probably address adding the test on its own ticket.

Reply:
yes, I believe it is sensible to have always build & test. And run these in release pipelines such that we can create a badge for this